wit_dashboard/wit-dashboard.html

```diff
@@ -8076,16 +8076,19 @@ <h2>Create similarity feature</h2>
           return 'inference_';
         },
 
-        makeUrl_: function(prefix, paramsDict) {
-          const url = prefix;
-          if (paramsDict) {
-            prefix +=
-              '?' +
-              Object.keys(paramsDict)
-                .map((k) => k + '=' + encodeURIComponent(paramsDict[k]))
-                .join('&');
+        makeUrl_: function(endpoint, paramsDict) {
+          // Strip off any leading slash to ensure relative path.
+          if (endpoint[0] === '/') {
```

Comment:
This is fragile but for regular usage of / in the path then it is enough (e.g. not for security).